tools/run_tests/run_microbenchmark.py

```diff
@@ -72,12 +73,16 @@ def link(txt, tgt):
   global index_html
   index_html += "<p><a href=\"%s\">%s</a></p>\n" % (tgt, txt)
 
-benchmarks = []
-profile_analysis = []
-cleanup = []
+def text(txt):
+  global index_html
+  index_html += "<p><pre>%s</pre></p>" % txt
```

Comment:
small nit: add a newline at end of `"<p><pre>%s</pre></p>"`? To `"<p><pre>%s</pre></p>\n"`?